pkg/bitcoin/electrum/electrum_integration_test.go

```diff
@@ -295,6 +301,67 @@ func TestGetBlockHeader_Negative_Integration(t *testing.T) {
 	}
 }
 
+func TestGetTransactionMerkleProof_Integration(t *testing.T) {
+	transactionHash := testData.TxMerkleProof.TxHash
+	blockHeight := testData.TxMerkleProof.BlockHeigh
+
+	expectedResult := &testData.TxMerkleProof.MerkleProof
+
+	for testName, config := range testConfigs {
+		t.Run(testName, func(t *testing.T) {
+			electrum := newTestConnection(t, config.clientConfig)
+
+			result, err := electrum.GetTransactionMerkleProof(
+				transactionHash,
+				blockHeight,
+			)
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			if diff := deep.Equal(result, expectedResult); diff != nil {
+				t.Errorf("compare failed: %v", diff)
+			}
+		})
+	}
+}
+
+func TestGetTransactionMerkleProof_Negative_Integration(t *testing.T) {
+	incorrectTransactionHash, err := bitcoin.NewHashFromString(
+		"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+		bitcoin.InternalByteOrder,
```

Comment:
I know it doesn't matter for the test outcome, but I'd prefer to use `bitcoin.ReversedByteOrde` for consistency with other `_Negative_Integration` tests.

Reply:
Done in 2d285459971908127d3d439ab2ed4fafa6f65b34.